packages/web-components/src/components/va-minimal-header/va-crisis-line-modal.tsx

```diff
@@ -0,0 +1,112 @@
+import { Component, Host, State, h } from '@stencil/core';
+import arrowRightSvg from '../../assets/arrow-right-white.svg';
+import { CONTACTS } from '../../../../react-components/src/components/Telephone/contacts';
+
+@Component({
+  tag: 'va-crisis-line-modal',
+  styleUrl: 'va-crisis-line-modal.scss',
+  shadow: true,
+})
+export class VACrisisLineModal {
+  @State() isOpen: boolean = false;
+  
+  setVisible() {
+    this.isOpen = true;
+  }
+
+  setNotVisible() {
+    this.isOpen = false;
+  }
+  
+
+  render() {
+    return (
+      <Host>
+        <div class="va-crisis-line-container">
+          <button onClick={()=>this.setVisible()} data-show="#modal-crisisline" class="va-crisis-line va-overlay-trigger">
+            <div class="va-crisis-line-inner">
+              <span class="va-crisis-line-icon" aria-hidden="true"></span>
+              <span class="va-crisis-line-text" >Talk to the <strong>Veterans Crisis Line</strong> now</span>
+              <img class="va-crisis-line-arrow" src={arrowRightSvg} aria-hidden="true"/>
+            </div>
+          </button>
+        </div>
+        <va-modal
+          modalTitle="We’re here anytime, day or night – 24/7"
+          onPrimaryButtonClick={() => this.setNotVisible()}
+          onCloseEvent={() => this.setNotVisible()}
+          visible={this.isOpen}
+        >
+          <div
+      id="modal-crisisline"
+      class="va-overlay va-modal va-modal-large"
+      role="alertdialog"
+    >
+      <div class="va-crisis-panel va-modal-inner">
+        <div class="va-overlay-body va-crisis-panel-body">
+          <p>
+            If you are a Veteran in crisis or concerned about one, connect with
+            our caring, qualified responders for confidential help. Many of them
+            are Veterans themselves.
+          </p>
+          <ul class="va-crisis-panel-list">
+            <li>
+              <i
+                class="fa fa-phone va-crisis-panel-icon"
+                aria-hidden="true"
+              />
+              {
+                // eslint-disable-next-line @department-of-veterans-affairs/prefer-telephone-component
```

Comment:
I'm guessing this component was just a lift and shift from vets website? We can probably clean up things like this though.

Reply:
Removed